_episodes/09-branches.md

```diff
@@ -87,60 +69,46 @@ one development line (branch) and this branch is called "master":
 
 ![]({{ site.baseurl }}/img/gitink/git-branch-1.svg)
 
-```shell
-$ git log --stat
-
-commit 7f3582dfbad6539cfa60f5b21bfad41d1b58a618
-Author: Radovan Bast <bast@users.noreply.github.com>
-Date:   Fri May 5 12:49:45 2017 +0200
-
-    do not forget to enjoy
-
- instructions.txt | 1 +
- 1 file changed, 1 insertion(+)
-
-commit 64441c1934def7d91ff0b66af0795749d5f1954a
-Author: Radovan Bast <bast@users.noreply.github.com>
-Date:   Fri May 5 12:49:35 2017 +0200
+We have defined the `git graph` alias earlier using:
 
-    add onion
-
- ingredients.txt | 1 +
- 1 file changed, 1 insertion(+)
+```shell
+$ git config --global alias.graph "log --all --graph --decorate --oneline --abbrev-commit"
+```
 
-commit d619bf848a3f83f05e8c08c7f4dcda3490cd99d9
-Author: Radovan Bast <bast@users.noreply.github.com>
-Date:   Fri May 5 12:48:36 2017 +0200
+Let us inspect the project history using the `git graph` alias:
 
-    adding ingredients and instructions
+```shell
+$ git graph
 
- ingredients.txt  | 3 +++
- instructions.txt | 5 +++++
- 2 files changed, 8 insertions(+)
+* dd4472c (HEAD -> master) we should not forget to enjoy
+* 2bb9bb4 add half an onion
+* 2d79e7e adding ingredients and instructions
 ```
 
 - Commits are states characterized by a 40-character hash (checksum).
+- `git graph` print abbreviations of these checksums.
 
 
 ### Branches are pointers
 
 - Branches are pointers that point to a commit.
-- Branch `master` points to commit `7f3582dfbad6539cfa60f5b21bfad41d1b58a618`.
+- Branch `master` points to commit `dd4472c8093b7bbcdaa15e3066da6ca77fcabadd`.
 
 Try this:
 
 ```shell
 $ cat .git/refs/heads/master
```

Comment:
Do we need this "under the hood" example?  We go over it later, for now I'm happy with the `git graph` output showing branch = pointer to some commit (mirrored on the board) seems good enough for now.

Reply:
Good point - I will get rid of it.